modules/seqexec/server/src/main/scala/seqexec/server/altair/AltairEpics.scala

```diff
@@ -176,9 +175,9 @@ class AltairEpics[F[_]: Async](service: CaService, tops: Map[String, String]) {
 object AltairEpics extends EpicsSystem[AltairEpics[IO]] {
 
   override val className: String = getClass.getName
-  override val Log: Logger = getLogger
   override val CA_CONFIG_FILE: String = "/Altair.xml"
 
-  override def build(service: CaService, tops: Map[String, String]) = new AltairEpics(service, tops)
+  override def build[F[_]: Sync](service: CaService, tops: Map[String, String]):F[AltairEpics[IO]] =
```

Comment:
Same here, why `AltairEpics[IO]` rather than `AltairEpics[F]`

Reply:
This is related to the previous question. `AltairEpics[IO]` is the type parameter of the trait implemented by the object where `build` is defined.